.github/workflows/node-ci.yml

```diff
@@ -1,15 +1,10 @@
 name: Node-CI
 
-on:
-  push:
-    branches: [master]
-  pull_request:
-    branches: [master]
+on: push
```

Comment:
This was previously setup just to run CI on pushes to master, not sure why. It now runs CI on all pushes to this repo.